src/privacy_accounting/optimizer/dp_optimizer.py

```diff
@@ -4,19 +4,19 @@
 import src.utils.torch_nest_utils as nest
 from src.privacy_accounting.analysis import QueryWithLedger
 from src.privacy_accounting.dp_query import GaussianDPQuery
-from . import WrapperOptimizer
+from .wrapper_optimizer import WrapperOptimizer
 
 
 class DPOptimizer(WrapperOptimizer):
 
     def __init__(self,
-                 optimizer,
+                 optimiser,
```

Comment:
omg all of these z have become an s!! pls no !! we are using z everywhere atm

Reply:
Yeah I think this is from the silly number of branches open

Reply:
Have fixed project wide I hope